mcs/class/System/System.Net.Sockets/Socket.cs

```diff
@@ -86,8 +86,8 @@ public partial class Socket : IDisposable
 		 */
 		internal EndPoint seed_endpoint = null;
 
-		internal Queue<SocketAsyncWorker> readQ = new Queue<SocketAsyncWorker> (2);
-		internal Queue<SocketAsyncWorker> writeQ = new Queue<SocketAsyncWorker> (2);
+		internal Queue<KeyValuePair<IntPtr, IOSelectorJob>> readQ = new Queue<KeyValuePair<IntPtr, IOSelectorJob>> (2);
```

Comment:
I don't understand the change here. When will the socket handle change between the many entries here?